### ISSUE TITLE
Directly read input.nml in CATChem

### ISSUE THREAD

**Issue by jianheACM** [ufs-community/CATChem#8]
Instead of passing individual namelist options from ccpp, we can pass the entire input.nml (containing CATChem namelist options) into CATChem.

**Comment by bbakernoaa** [comment]
To give a little more context... 

Right now the CATChem namelist values in the FV3 `input.nml` file are located under the physics group.  This will move the CATChem variables to its own group with the flag in physics cplchp set to turn to turn on CATChem

For example

```fortran 
&gfs_physics_nml 
  ...
  cplchp=.true.
/

&catchem_nml
  ...
/
```

**Comment by jianheACM** [comment]
@bbakernoaa Yes Barry. We need a new namelist entry for CATChem. I also would like to have input.nml read directly in CATChem folder. This would make things easier if we want to include additional namelist options in the future.
